test/genstress.proj

```diff
@@ -90,17 +88,32 @@
     <WriteLinesToFile File="$(UnitTestExtractedSemaphore)" Lines="" Overwrite="false" />
   </Target>
  
-  <Target Name="UnzipTest"
+    <Target Name="UnzipTest"
     Inputs="@(ZipFile)"
-    Outputs="@(ZipFile->Replace('$(UnitTestDirectory)','$(UnitTestExtractDir)'))">
+    Outputs="@(ZipFile->Replace('$(UnitTestDirectory)','$(UnitTestExtractDir)'))"
+    DependsOnTargets="SelectTestZips">
+        <Message Text="[REMOVE ME] This is @(ZipFile)" />
+        
+        <ZipFileExtractToDirectory Condition="!Exists('%(RootDir)%(Directory)%(ZipFile.FileName)')"
+            SourceArchive="%(ZipFile.FullPath)"
+            DestinationDirectory="$(UnitTestExtractDir)\%(filename)%(extension)"
+            OverwriteDestination="true" />
+        
+        <Copy SourceFiles="@(JsonFile)" DestinationFiles="@(JsonFile->Replace('$(UnitTestDirectory)','$(UnitTestExtractDir)'))" />
+    </Target>
+
+    <Target Name="SelectTestZips">
+        <Error Condition="!Exists('$(MSBuildThisFileDirectory)\..\AnyOS.AnyCPU.Debug-Test-Archive-List.txt')" Text="Can't find AnyOS Test List.'" />
```

Comment:
This is placed here temporarily. Waiting on CoreFX tests to come back online to complete this piece.